recipes/xorg-gccmakedep/all/conanfile.py

```diff
@@ -0,0 +1,84 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+from conans.errors import ConanInvalidConfiguration
+import os
+import re
+
+required_conan_version = ">=1.33.0"
+
+
+class XorgGccmakedep(ConanFile):
+    name = "xorg-gccmakedep"
+    description = "script to create dependencies in makefiles using 'gcc -M'"
+    topics = ("conan", "xorg", "gcc", "dependency", "obsolete")
```

Comment:
```suggestion
    topics = ("xorg", "gcc", "dependency", "obsolete")
```

Don't add the Conan tag for new recipes anymore please